Source/ACE.Server/Command/Handlers/AdminCommands.cs

```diff
@@ -1181,14 +1181,14 @@ public static void Handledirect_emote_select(Session session, params string[] pa
         }
 
         // dispel
-        [CommandHandler("dispel", AccessLevel.Developer, CommandHandlerFlag.RequiresWorld, 0)]
+        [CommandHandler("dispel", AccessLevel.Developer, CommandHandlerFlag.None, 0, "Removes all enchantments from the player", "dispel")]
```

Comment:
I think this should still be RequiresWorld.

You wouldn't be able to do it while in portal space, but, None would allow this from the console I believe. Have to confirm this.